lnwire/onion_error.go

```diff
@@ -78,6 +79,7 @@ const (
 	CodeFinalIncorrectCltvExpiry         FailCode = 18
 	CodeFinalIncorrectHtlcAmount         FailCode = 19
 	CodeExpiryTooFar                     FailCode = 21
+	CodeInvalidOnionPayload                       = FlagPerm | 22
```

Comment:
I still don't understand the `Perm` idea. To me it is similar to the odd/even situation: info that you don't need.

Reply:
not sure, it predates my involvement :)